orc8r/cloud/go/tools/combine_swagger/testdata/configs/importer.yml

```diff
@@ -4,7 +4,7 @@ swagger: '2.0'
 magma-gen-meta:
   go-package: magma/orc8r/cloud/go/tools/swaggergen/testdata/importer/models
   dependencies:
-    - 'orc8r/cloud/go/tools/swaggergen/testdata/base.yml'
+    - 'configs/base.yml'
```

Comment:
So can you clarify why we're keeping `configs/` prefix, was unclear to me

Reply:
Notice that `orc8r/cloud/go/tools/swaggergen/testdata/base.yml` is in the `swaggergen` package whereas this file, `importer.yml` is in the `combine_swagger` package. `importer.yml` was referencing a dependency in the `swaggergen` package even though the same file was available in `combine_swagger/testdata/configs`. Since `rootDir` is `combine_swagger/tesdata`, the `configs` prefix is needed.